src/components/widgets/Accordion.js

```diff
@@ -46,7 +46,7 @@ export default function Accordion() {
     <section className="wd-content" id="toc-accordion">
       <h2 id="accordion" className="section-top anchor">Accordion</h2>
       <p>Use the <strong>Accordion</strong> to display content in collapsible compartments.</p>
-      <p>Use <span className="wd-monospace">&lt;div class="accordion"&gt;&lt;/div&gt;</span> to wrap your content.</p>
+      <p>Wrap your full accordion with <span className="wd-monospace">&lt;div class="accordion"&gt;&lt;/div&gt;</span>.<br/> Then, wrap each tab (and its collapsible content) with <span className="wd-monospace">&lt;div class="accordion-item"&gt;&lt;/div&gt;</span>.</p>
```

Comment:
Please explain that the first heading within accordion-item serves as the accordion title.